scripts/map.js

```diff
@@ -575,14 +575,28 @@ function Map(display, __game) {
     /* canvas-related stuff */
 
     this.getCanvasContext = wrapExposedMethod(function() {
-        return $('#drawingCanvas')[0].getContext('2d');
+        var ctx = $('#drawingCanvas')[0].getContext('2d');
+        if(!this._dummy) {
```

Comment:
Text drawn by the canvas should be in the same font as other text drawn by the game.